app/moves/middleware.js

```diff
@@ -116,24 +115,52 @@ const movesMiddleware = {
           return moveService
             .getMovesCount({ dateRange, status: moveType.filter, locationId })
             .then(count => {
-              const { label } = moveType
               return {
-                label,
+                ...moveType,
                 value: count,
                 active: moveType.filter === req.params.status,
                 href: `${req.baseUrl}/${period}/${date}${
                   locationId ? '/' + locationId : ''
                 }/${moveType.filter}`,
               }
             })
-            .then(presenters.moveTypesToFilterComponent)
         })
       )
       next()
     } catch (error) {
       next(error)
     }
   },
+  setTotalMoves: (req, res, next) => {
```

Comment:
I wonder if this should actually be in that dashboard controller?

The middleware functions are useful for things that are shared across lots of routes, but this is only used by the dashboard right?

Reply:
For now yes, and indeed it's written in such a fashion that it's not really reusable much outside the OCA journey. I'll move it.